hosted/helpers/helper_common.go

```diff
@@ -202,3 +205,38 @@ func GetCommonMetadataLabels() map[string]string {
 		"testfilenumber": filename,
 	}
 }
+
+func GetK8sVersion(provider string) string {
+	k8sVersion := os.Getenv("KUBERNETES_VERSION")
+	if k8sVersion != "" {
+		return k8sVersion
+	}
+	switch provider {
+	case "gke":
```

Comment:
For GKE, we would also have to set k8s version for nodepool in addition to  cluster k8s version
https://github.com/rancher/hosted-providers-e2e/blob/main/cattle-config-provisioning.yaml#L116

Reply:
We are not setting the k8s version here, we are simply reading it. But I think this is a good catch, because currently this env var is only taken into account by import tests, we'll need to modify the provisioning tests to take into account this env var.

Reply:
Ok, please create an issue for this

Reply:
There's no need to create a separate issue, I have implemented it in this PR.
See [8fce45d](https://github.com/rancher/hosted-providers-e2e/pull/41/commits/8fce45d1ca999697fbc5f31814c117d1f0db53d6).